src/main/java/seedu/guest/MainApp.java

```diff
@@ -138,26 +138,26 @@ protected Config initConfig(Path configFilePath) {
      * or a new {@code UserPrefs} with default configuration if errors occur when
      * reading from the file.
      */
-    protected UserPrefs initPrefs(UserPrefsStorage storage) {
-        Path prefsFilePath = storage.getUserPrefsFilePath();
+    protected GuestPrefs initPrefs(GuestPrefsStorage storage) {
```

Comment:
Sharing similar opinions with Bryan, should we keep them as UserPrefs since it is the "user" that will be using it?